astro/src/content/docs/_shared/_licensed-edition-blurb-api.astro

```diff
@@ -0,0 +1,5 @@
+---
+const { feature } = Astro.props;
+---
+{feature && <><p><strong>Note:</strong> A license is required to utilize {feature}.</p></>}
```

Comment:
fairly minor complaint, but we have had some discussions around aiming to use markdown wherever possible. In this case you could convert this to a one-line mdx file:

`**Note:** A license is required to utilize {props.feature}`